docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :release:`1.28.0`
+* :feature:`5569` Make it possible for the users to initiate evm accounts detection and add a periodic task to do it automatically.
```

Comment:
A user does not understand what this means. This is addressed to a user, not a dev.

```suggestion
* :feature:`5569` It is now possible to detect when an evm address is used in other chains and start tracking it. Also a background task runs periodically that performs this check.
```